### PR TITLE
Refactoring/spk-119. Organise the code packages per features as opposed as per android component 

### PR DESCRIPTION
**Pull request by AlexMisiulia**

### CODE REVIEW

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseFragment.kt [hunk 1]
```diff
@@ -51,7 +52,12 @@ class CreateFormBaseFragment : BaseFragment() {
         }
 
         viewModel.continueLiveData.observeEvent(this) {
-            val detailsFragment = CreateFormDetailsFragment.newInstance(it.formName, it.projectName, it.keyExpiryDate)
+            val detailsFragment =
```
Comment (AlexMisiulia) [inline app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseFragment.kt]: Fix formatting

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseViewModel.kt [hunk 2]
```diff
@@ -49,13 +51,19 @@ class CreateFormBaseViewModel @Inject constructor(
             dateUtils.parse(currentKeyExpiryText, DATE_FORMAT_DD_MM_YY)
         }
 
-        val datePickerInfo = DatePickerInfo(minDate = Date(), displayDate = dateToDisplay)
-        _showDatePickerLiveData.value = Event(datePickerInfo)
+        val datePickerInfo =
```
Comment (AlexMisiulia) [inline app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseViewModel.kt]: Fix formatting

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseViewModel.kt [hunk 3]
```diff
@@ -49,13 +51,19 @@ class CreateFormBaseViewModel @Inject constructor(
...
+                displayDate = dateToDisplay
+            )
+        _showDatePickerLiveData.value =
+            Event(datePickerInfo)
     }
 
     fun onContinueClicked(projectName: String, formName: String, keyExpiryDate: String) {
         if (formName.isEmpty()) {
-            _formNameValidationErrorLiveData.value = Event(R.string.form_name_must_be_not_empty)
+            _formNameValidationErrorLiveData.value =
```
Comment (AlexMisiulia) [inline app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseViewModel.kt]: Fix formatting

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseFragment.kt (hunk 1 above)
Comment (AlexMisiulia) [reply app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseFragment.kt]: Created another PR for code formatting: https://github.com/lampkicking/social-impact-keys/pull/43

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseViewModel.kt (hunk 2 above)
Comment (AlexMisiulia) [reply app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseViewModel.kt]: Created another PR for code formatting: https://github.com/lampkicking/social-impact-keys/pull/43

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseViewModel.kt (hunk 3 above)
Comment (AlexMisiulia) [reply app/src/main/java/com/yoti/mobile/android/social/keys/features/createform/CreateFormBaseViewModel.kt]: Created another PR for code formatting: https://github.com/lampkicking/social-impact-keys/pull/43

**Comment by AlexMisiulia** [comment]
@cquemin during moving classes to other packages AndroidStudio suddenly change the formatting to ugly :( I created another pr https://github.com/lampkicking/social-impact-keys/pull/43 to fix formatting in the whole project according to kotlin convention.

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/shared/features/formlist/FormAdapter.kt [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package com.yoti.mobile.android.social.keys.adapter
+package com.yoti.mobile.android.social.keys.features.shared.features.formlist
```
Comment (cquemin) [inline app/src/main/java/com/yoti/mobile/android/social/keys/features/shared/features/formlist/FormAdapter.kt]: Features twice ?

Comment (AlexMisiulia) [reply app/src/main/java/com/yoti/mobile/android/social/keys/features/shared/features/formlist/FormAdapter.kt]: @cquemin just my vision. This is the package for screens (Fragments) that can't be reused in different Activities.

**Pull request merged**